src/rpcserver.cpp

```diff
@@ -399,6 +399,7 @@ static const CRPCCommand vRPCCommands[] =
     /* Omni Core hidden calls - development usage (not shown in help) */
     /* CATEGORY                              NAME                               ACTOR (FUNCTION)                  OKSAFEMODE  THREADSAFE  REQWALLET */
     { "hidden",                              "mscrpc",                          &mscrpc,                          true,       true,       true },
+    { "hidden",                              "omni_activatefeature",            &omni_activatefeature,            true,       true,       true },
```

Comment:
Hm. I should rename it to `omni_sendactivation` or so, to be consistent with the other `omni_sendxxxxx` calls. At some point there might be `omni_createxxxx`.